modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -523,6 +526,281 @@ protected function register_controls() {
 		] );
 
 		$this->end_controls_section();
+
+		$this->start_controls_section( 'section_title_style', [
+			'label' => esc_html__( 'Titles', 'elementor' ),
+			'tab' => Controls_Manager::TAB_STYLE,
+		] );
+
+		$this->start_controls_tabs( 'title_style' );
+
+		$this->start_controls_tab(
+			'title_normal',
+			[
+				'label' => esc_html__( 'Normal', 'elementor' ),
+			]
+		);
+
+		$this->add_group_control( Group_Control_Typography::get_type(), [
+			'name' => 'title_typography',
+			'global' => [
+				'default' => Global_Typography::TYPOGRAPHY_ACCENT,
+			],
+			'fields_options' => [
+				'font_family' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-font-family: "{{VALUE}}";',
+					],
+				],
+				'font_size' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-font-size: {{SIZE}}{{UNIT}};',
+					],
+				],
+				'font_weight' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-font-weight: {{VALUE}};',
+					],
+				],
+				'text_transform' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-text-transform: {{VALUE}};',
+					],
+				],
+				'font_style' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-font-style: {{VALUE}};',
+					],
+				],
+				'text_decoration' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-text-decoration: {{VALUE}};',
+					],
+				],
+				'line_height' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-line-height: {{SIZE}}{{UNIT}};',
+					],
+				],
+				'letter_spacing' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-letter-spacing: {{SIZE}}{{UNIT}};',
+					],
+				],
+				'word_spacing' => [
+					'selectors' => [
+						'{{WRAPPER}}' => '--tabs-v2-title-typography-word-spacing: {{SIZE}}{{UNIT}};',
+					],
+				],
+			],
+		] );
```

Comment:
I like the creativity here and it would work, but I prefer to leave it with a simple selector.
The reason: If at any point we would want to add another control to the Typography group control (e.g. `text-stroke`), we would have to go to all the places that implement individual CSS variable selectors and add that control to all of them.

A single selector removes the need for that, at the price of not supporting CSS variables, I'm aware of that.

Reply:
updated. 

Credits to Leonid! He had done this setup.